src/docs/integrations/vercel.mdx

```diff
@@ -6,20 +6,32 @@ title: "Vercel Integration"
 
 To use the Vercel integration you'll need to create your own integration in Vercel. To start, click "Create" in the [Integrations Developer Console](https://vercel.com/dashboard/integrations/console).
 
-When configuring the app:
-* Set the `Redirect URL` to `{YOUR_DOMAIN}/extensions/vercel/configure/`
-* Set the `Webhook URL` to `{YOUR_DOMAIN}/extensions/vercel/delete/` and subscribe to the following events:
-  * Deployment Created
-  * Configuration Removed
-* Leave the `Configuration URL` empty
+When configuring the app, under the **Settings** section:
 
-Take note of your client ID, client secret, and integration URL slug (the value you entered in the URL Slug field. This will be used to construct the integration installation URL). Add those to `config.yml` like this:
+- Set the `Redirect URL` to `{YOUR_DOMAIN}/extensions/vercel/configure/`
+- Set the following API Scopes:
+
+| API Scope                     | Value      |
+| ----------------------------- | ---------- |
+| Integration Configuration     | Read/Write |
+| Projects                      | Read/Write |
+| Project Environment Variables | Read/Write |
+| Teams                         | Read       |
+| Current User                  | Read       |
+
+- Set the `Webhook URL` to `{YOUR_DOMAIN}/extensions/vercel/delete/` and subscribe to the following events:
```

Comment:
This will change after we finish up that old webhook stuff, right? We want to use the "generic webhook" instead of the delete one, so maybe this PR should hang out for a bit until that change lands.

Reply:
That is true, I don't mind opening a separate PR for that though because currently, users integrating with Sentry will see these scopes and not know which to grant.